File: new SpringSecurity/SpringSecurity/src/main/java/com/example/demo/entities/Team.java
package com.example.demo.entities;
import java.sql.Date;

import jakarta.persistence.*;

@Entity
@Table(name="teams")
public class Team {
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	int teamid;
	
	@Column
	int pid;
	
	@Column
	int empid;
	
	@Column
	String comments;
	
	@Column
	Date assigneddate;
	
	@Column
	boolean status;
	
	@Column
	Date releasedate;
	
	
	
}

File: DotNetApi/DotNetWebApi/Models/Team.cs
using System;
using System.Collections.Generic;

namespace DotNetWebApi.Models
{
    public partial class Team
    {
        public int Teamid { get; set; }
        public int Pid { get; set; }
        public int Empid { get; set; }
        public string? Comments { get; set; }
        public DateOnly? AssignedDate { get; set; }
        public ulong? Status { get; set; }
        public DateOnly? Releasedate { get; set; }

        public virtual Employee Emp { get; set; } = null!;
        public virtual Project PidNavigation { get; set; } = null!;
    }
}
